version.yml

```
@@ -0,0 +1,8 @@
+graylog:
+  major_version: 4
+  minor_version: 1
+  patch_version: 9
+  release: 1
+forwarder:
+  version: 4.1
+  release: 5
```

I'm not 100% sure, but do we really have release `5` out there? The highest number I can find in an existing tag was `2`.

```suggestion
  release: 2
```

---

I was just copying what was in the `VERSION_FORWARDER_PACKAGE` file. But you're right, we only have rev 2 published.

---

Okay, I was wrong. We do have 4.1 rev 5. Because we had a beta and some release candidates.